app/models/post_object.rb

```diff
@@ -0,0 +1,147 @@
+class Post
```

Comment:
Hrm. You likely would have wanted to change the class name in this as well as the file name.